scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -26,6 +26,12 @@ namespace <%= namespace %> {
             }
         }
 
+        public List<UserError> UserErrors {
```

Comment:
This is true. But you also want to persist `UserErrors` instead of just an event/callback firing.